### ISSUE TITLE
Jack hammer sound in New York level

### ISSUE THREAD

**Issue by buvk** [PARTYMANX/partymod-thps2#12]
Not sure if this is a bug or not, but in the PC version of THPS2 (happens in vanilla PC as well), you can hear the jack hammer sound all over the place.  Pretty sure that in the PSX and DC versions, you can only really hear it if you are in the corner near the awnings.

Is this something that could potentially be fixed?

**Comment by PARTYMANX** [comment]
there is a non-positional jackhammer in the ambient track for the level (music/LTIX03.dat which can also be played in VLC), you might not remember it as that track only plays when music is not playing.  that being said there is a jackhammer-like sound from the train which is a strange error in the pc version.  i think it's replaying the train sound rapidly

**Comment by buvk** [comment]
Yeah that might be what I am referring to. I thought it was a taxi at first making it.

**Comment by buvk** [comment]
I deleted LTIX03.dat and turned off the music. The sound definitely happens when the train goes by.

**Comment by buvk** [comment]
The bus in Philly has the same issue.

**Comment by PARTYMANX** [comment]
I put together a fix for this in 1.1.0.  The issue was that the sound was being passed between vehicles if there was more than one instance of a vehicle with looping sound, and whenever that sound exited the range that you could hear, it would stop itself, so it was just constantly restarting each frame
